templates/taxi/car_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+    <h1>Delete Car</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <h3>You gonna delete {{ car.model }}. Are you sure?</h3>
+    <br>
+    <input type="submit" value="Submit" class="btn btn-danger">

```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on. Read checklist

Reply:
![image](https://github.com/mate-academy/py-taxi-service-forms/assets/62641628/f2b6fac4-4559-4a57-87c5-709ba97cf8c3)